packages/create-yoshi-app/templates/server/javascript/index.js

```diff
@@ -1,5 +1,6 @@
 const bootstrap = require('@wix/wix-bootstrap-ng');
 
 bootstrap()
+  .config('dist/src/config') // https://github.com/wix-platform/wix-node-platform/tree/master/bootstrap/wix-bootstrap-ng#wixbootstrapngconfigfileexportingfunction-this
```

Comment:
I know that it's rather opinionated, but we usually put comments above the lines

```suggestion
  // https://github.com/wix-platform/wix-node-platform/tree/master/bootstrap/wix-bootstrap-ng#wixbootstrapngconfigfileexportingfunction-this
  .config('dist/src/config')
```